utils/volumeutils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	config "github.com/spf13/viper"
+)
+
+func GetVolumeDir(volumeName string) string {
```

Comment:
Good idea. On the long term, I'd prefer `brick` package to not depend on `volume` package.